.github/component_owners.yml

```diff
@@ -9,6 +9,10 @@ components:
     - toddbaert
   src/OpenFeature.Contrib.Providers.GOFeatureFlag:
     - thomaspoignant
+  src/OpenFeature.Contrib.Providers.Flagsmith:
+    - vpetrusevici
+    - matthewelwell
+    - novakzaballa
```

Comment:
Not a blocker, but `novakzaballa` will not be added to reviews since they are not a member of the org. They can create a PR to add themselves [here](https://github.com/open-feature/community/blob/ab73778324513cc789f184408afcf2912fceec53/config/open-feature/org.yaml#L29).

Reply:
removed